### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ggabernet**
Prepare release 2.0

## PR checklist
- [ ] Bump all versions to 2.0, and changelog update date
- [ ] This comment contains a description of changes (with reason)
- [ ] `CHANGELOG.md` is updated
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] Documentation in `docs` is updated
- [ ] If necessary, also make a PR on the [nf-core/epitopeprediction branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/epitopeprediction)

### CODE REVIEW

File: workflows/epitopeprediction.nf [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+                netmhc_meta_data[it].software_md5,
+                file(params["${it}_path"], checkIfExists:true),
+                file(netmhc_meta_data[it].data_url),
+                netmhc_meta_data[it].data_md5,
+                netmhc_meta_data[it].binary_name
+            ])
+        }
+    }
+    // import external tools
+    EXTERNAL_TOOLS_IMPORT(ch_nonfree_paths)
```
Comment (ggabernet) [inline workflows/epitopeprediction.nf:239]:
```suggestion
    EXTERNAL_TOOLS_IMPORT(
        ch_nonfree_paths
    )
```

**Review by ggabernet — COMMENTED** [review]
Just went over it and it looks good to me (but can't approve as I'm the PR author).

Just had a small comment of formatting

File: lib/WorkflowEpitopeprediction.groovy [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+        yaml_file_text        += "section_name: '${workflow.manifest.name} Workflow Summary'\n"
+        yaml_file_text        += "section_href: 'https://github.com/${workflow.manifest.name}'\n"
+        yaml_file_text        += "plot_type: 'html'\n"
+        yaml_file_text        += "data: |\n"
+        yaml_file_text        += "${summary_section}"
+        return yaml_file_text
+    }
+
+    //
+    // Exit pipeline if incorrect --genome key provided
```
Comment (marissaDubbelaar) [inline lib/WorkflowEpitopeprediction.groovy:47]: Maybe remove this?

File: modules.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
+{
+    "name": "nf-core/epitopeprediction",
+    "homePage": "https://github.com/nf-core/epitopeprediction",
+    "repos": {
+        "nf-core/modules": {
+            "custom/dumpsoftwareversions": {
+                "git_sha": "20d8250d9f39ddb05dfb437603aaf99b5c0b2b41"
+            },
+            "fastqc": {
```
Comment (marissaDubbelaar) [inline modules.json:9]: We could also remove the fastqc related files, right?

**Review by marissaDubbelaar — APPROVED** [review]

File: subworkflows/local/input_check.nf [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    def extension = input_file.extension
+
+    if ( extension == "vcf" | extension == "vcf.gz" ) {
+        filetype = "variant"
+    }
+    else if ( extension == "tsv" | extension == "GSvar" ) {
+        // Check if it is a variant annotation file or a peptide file
+        input_file.withReader {
+            def first_header_col = it.readLine().split('\t')[0]
+            //first_header_col = [col.lower() for col in tsv.readlines()[0].split('\t')][0]
```
Comment (christopher-mohr) [inline subworkflows/local/input_check.nf:67]:
```suggestion
```

**Review by christopher-mohr — COMMENTED** [review]
Just found some minor things that could be changed.

File: .nf-core.yml [hunk 5]
```diff
@@ -0,0 +1,3 @@
+lint:
+    files_unchanged:
+      - docs/images/nf-core-epitopeprediction_logo_dark.png
```
[the remark this answers is not in the document]
Comment (ggabernet) [reply .nf-core.yml:3]: probably not if you've added. the dark logo like in the template

File: CITATIONS.md [hunk 6]
```diff
@@ -10,7 +10,6 @@
 
 ## Pipeline tools
 
-* [FastQC](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/)
```
Comment (christopher-mohr) [inline CITATIONS.md:13]:
```suggestion
```

File: modules.json (hunk 3 above)
Comment (ggabernet) [reply modules.json:9]: done in a separate PR

File: .nf-core.yml (hunk 5 above)
[the remark this answers is not in the document]
Comment (ggabernet) [reply .nf-core.yml:3]: removed !

**Review by christopher-mohr — APPROVED** [review]
LGTM

**Review by christopher-mohr — APPROVED** [review]

**Pull request merged**